src/styles/partials/components/_alert.scss

```diff
@@ -0,0 +1,115 @@
+.dg_alert {
+  background: $vibrant-blue;
+  padding: $default-padding;
+
+  .dg_alert__container {
+    display: flex;
+    flex-wrap: nowrap;
+    align-items: center;
+
+    margin: auto;
+    max-width: 1400px;
+    width: 100%;
+  }
+
+  * {
+    color: $white;
+  }
+
+  a {
+    text-decoration: underline;
+
+    &:hover {
+      background: $white;
+      color: $vibrant-blue;
+    }
+  }
+
+  p {
+    font-size: $smallest-heading-font-size;
+    margin-bottom: 0;
+  }
+
+  p,
+  .dg_alert__status {
+    font-family: $default-heading-font;
+  }
+
+  .dg_alert__status {
+    font-weight: $font-bold;
+    text-transform: uppercase;
+  }
+
+  .dg_alert__icon {
+    font-size: $large-icon-size;
+    padding: 0 ($default-padding * 2) 0 $default-padding;
+  }
+
+  &.emergency {
+    background: $error-color;
+
+    a:hover {
+      background: $white;
+      color: $error-color;
+    }
+  }
+
+  &.status {
+    border: $border-width-small solid $white;
+    outline: $border-width-large solid transparent;
+
+    * {
+      color: $white;
+    }
+
+    &.success {
+      background: $success-color;
+      border-color: $black;
+      outline-color: $success-color;
+
+      * {
+        color: $black;
+      }
+
+      a:hover {
+        background: $black;
+        color: $success-color;
+      }
+    }
+
+    &.info {
+      background: $info-color;
+      outline-color: $info-color;
+
+      a:hover {
+        background: $white;
+        color: $info-color;
+      }
+    }
+
+    &.warning {
+      background: $warning;
+      border-color: $black;
+      outline-color: $warning;
+
+      * {
+        color: $black;
+      }
+
+      a:hover {
+        background: $black;
+        color: $warning-color;
+      }
+    }
+
+    &.error {
+      background: $error-color;
+      outline-color: $error-color;
+
+      a:hover {
+        background: $white;
+        color: $error-color;
+      }
+    }
+  }
+}
```

Comment:
Default and Emergency alert items font and font style appear correct. All other alert subtext should be italic and `"Crimson Text", serif`. @sdurphy @martypowell

Reply:
thanks @theilman7 checking shortly.

Reply:
confirmed, and updated.

Reply:
I think this is bold italic from the looks of it in the XD.

Reply:
Looks like the subtext has been set to italic, so that looks good. Still seeing the `font-family` of the subtext set to `"Montserrat", sans-serif; `vs. `"Crimson Text", serif` though.